.coveragerc

```diff
@@ -17,13 +17,11 @@ omit =
     */flows/classic.py
     */flows/misc.py
     */flows/optimizing.py
-    */steps/checker.py
```

Comment:
not in the changelog

Reply:
Expanded coverage for openlane/steps/checker.py

Reply:
ohhh these are the ignored list. so you don't ignore them anymore 👍🏿